system/jlib/jstats.cpp

```diff
@@ -1441,7 +1441,8 @@ bool StatsScopeId::setScopeText(const char * text, const char * * _next)
         if (MATCHES_CONST_PREFIX(text, FunctionScopePrefix))
         {
             setFunctionId(text+ strlen(FunctionScopePrefix));
-            *_next = text + strlen(text);
+            if (next)
```

Comment:
It's probably ok as I think they always point to the same place, but would seem more logical to say 

if (_next) 

here?

Reply:
I agree